packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -55,6 +55,10 @@ const ConnectorDetails = () => {
   const isSocial = data?.type === ConnectorType.Social;
   const [isDeleteAlertOpen, setIsDeleteAlertOpen] = useState(false);
 
+  useEffect(() => {
+    setIsDeleteAlertOpen(false);
+  }, [connectorId]);
```

Comment:
It might be more reliable to detect route change instead of connectorId change. Same as #2750

Reply:
updated.